### DISCUSSION TITLE
STT Whisper not working

### DISCUSSION THREAD

**Discussion by karl0ss** [open-webui/open-webui#3302] [Q&A] [answered]
Hope the community can help me with this wonderful app..

Been running for awhile now (using offical docker image, with my nvidia gpu's passed thuogh)

I went to try the "call" option in the later releases and after it looked like the bot was listening to me, it returned the errror ```Requested int8 compute type, but the target device or backend do not support efficient int8 computation.```
![image](https://github.com/open-webui/open-webui/assets/2493260/fb46c1fa-fe3f-4903-a425-0eb64aeb5272)

I don't seem to see this around anywhere, or how to fix it, so I thought I would ask, i'm running Nvidia Tesla m60

![image](https://github.com/open-webui/open-webui/assets/2493260/5874ea76-128b-497d-bf3a-528d262bafe3)

**Comment by justinh-rahb** [comment]
You must be using the `:cuda` image. Maxwell is pretty old and unlikely to be supported by most ML libraries anymore, you'd likely actually get better performance in CPU mode anyway. FWIW the CUDA image is only accelerating the RAG embedding/reranking models and Whisper TTS, they're small enough to run on CPU without much appreciable difference in speed.

**Reply by karl0ss** [reply]
Ah, Amazing, I am indeed running the cuda image, I will swtich to the normal image, thank you for your help.

**Reply by karl0ss** [reply, marked as answer]
Can confirm it is working fine with the standard image, thanks for your help!
